packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -55,7 +55,7 @@
           </div>
           <div
             class="font-medium pt-2 text-indigo-600 text-18px leading-28px"
-            :class="{ 'text-jade-600': browser.isSelected, 'text-gray-500': browser.disabled || !browser.isVersionSupported }"
+            :class="{ 'text-jade-600': browser.id === selectedBrowserId, 'text-gray-500': browser.disabled || !browser.isVersionSupported }"
```

Comment:
Updated this to ensure we're always reflecting the currentBrowser value in our selections.

Reply:
Could we not have used the `checked` slot prop? We've got so many conditionals in this template 😅

Reply:
Ah good call, that works great. I'll have to do some reading on how the slot props work.